### ISSUE TITLE
MapViewLite goes to black after resuming from another map activity

### ISSUE THREAD

**Issue by shamilds** [heremaps/here-sdk-examples#26]
I have MainActivity with MapViewLite view, everything works as expected. If I move to a different activity which as another MapViewLite view, second activity displays map fine. But coming back to MainActivity, MapViewLite view turns black. I can reproduce this issue with sdk sample. I am testing on Samsung Note5 and Samsung S10, both has the same issue. No exceptions on logcat either.

This happens only after displaying second MapViewLite view. Tried few things and went through sdk documentation, could not find anything related to this. Is there anything additional I have to do before or after displaying second MapViewLite?

sdk version : heresdk-lite-android-4.3.2.0.1948.aar
sample app : https://github.com/shamilds/here-sample

Any help would appreciate.

NOTE: I accidentally posted this on v3.x repository, @NazarKacharaba pointed out I am on wrong repository. As I can remember I had same issue with v3.x but now upgraded to v4.x which is much easy to handle, but this issue not helping.

**Comment by datasun** [comment]
Thanks for your question. This is not an issue of this repo or the hosted example apps.

The HERE SDK for Android (Lite Edition) supports only _one_ `MapViewLite` instance at a time. In your example it looks like you are creating more than one instance. This is mentioned in the [Get Started](https://developer.here.com/documentation/android-sdk-lite/4.3.2.0/dev_guide/content/topics/quick-start.html#add-the-map-view) guide.

Only the HERE SDK for Android (_Explore Edition_ or _Navigate Edition_) support multiple map view instances.

Hope this fixes your issue. If not, please leave a comment or raise a new issue. Thanks.
